deploy/helm/sumologic/values.yaml

```diff
@@ -165,6 +165,24 @@ sumologic:
     ## Option to specify the Fluentd buffer as file/memory.
     buffer: "memory"
 
+    securityContext:
+      fsGroup: 999
```

Comment:
FYI: This is the fluentd user id. By default pvc is always mounted as root user so without setting the security context the fluentd user does not have permission to write to the mounted volume. I spent a long time trying to figure out why the fluentd chart does not need to set this. The fluentd dockerfile [link](https://github.com/fluent/fluentd-docker-image/blob/master/v1.6/debian/Dockerfile#L49) says that everything under the `/fluentd` folder should be owned by the fluentd user, but it doesn't work in our case even though our image is built on top of fluentd's. 

I looked at the helm stable repo and saw a lot of charts setting the security context fsGroup, it looks like it's very common so I'm following this pattern.

Reply:
should we add a `// NOTE` as well explaining `999` ?